packages/eslint-config-shakacode/rules/javascript.yml

```diff
@@ -17,3 +17,18 @@ rules:
     - 2
     - args: "all"
       argsIgnorePattern: ^_
+
+  # http://eslint.org/docs/rules/max-len
+  # https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L128
+  # Almost identical to airbnb's version, but adds ignore of `import` statements
+  max-len:
+    - 2
+    -
+      code: 100
+      tabWidth: 2
+      ignoreUrls: true
+      ignoreComments: false
+      ignoreRegExpLiterals: true
+      ignoreStrings: true
+      ignoreTemplateLiterals: true
+      ignorePattern: ^import\\s+
```

Comment:
I never knew that Airbnb had ignore strings and template literals on lol

Are we sure we want to do this instead of:

```js
import {
  foo,
  bar,
} from './someLongName';
```

I think it's important to see where things are coming from. In our internal project, we can use aliases to keep the `from ''` part short as well as refactoring our `libs` to be less nested.